devchat.go

```diff
@@ -591,27 +592,32 @@ func runCommands(line string, u *user, isSlack bool) {
 		return
 	}
 	if strings.HasPrefix(line, "/hang") {
-		rest := strings.TrimSpace(strings.TrimPrefix(line, "/hang"))
-		if len(rest) > 1 {
-			u.writeln(u.name, line)
-			hangGame = &hangman{rest, 15, " "} // default value of guesses so empty space is given away
+		if !hangGameStarted {
+			word, err := u.term.ReadPassword("Word (Won't be displayed): ")
```

Comment:
Using ReadPassword will turn off echo, which can be confusing to people not so used to the command line.

Reply:
Yes I'm aware of that, it is the reason I added `(Won't be displayed)`. But doesn't matter any more since this change is not needed.